Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -439,7 +439,7 @@ extension ResourceFileElements {
             },
             excluding: excluding.map {
                 let excludePath = path.appending(RelativePath($0))
-                let excludeGlob = excludePath.extension != nil ? excludePath : excludePath.appending(component: "**")
+                let excludeGlob = (excludePath.extension != nil && absolutePath.pathString.suffix(3) != "/**")) ? excludePath : excludePath.appending(component: "**")
```

Comment:
AFAIK SPM doesn't support `/**` but it just handles directories, why do we need to change this logic?

Reply:
it's only a string, better to check then to fail..

Reply:
otherwise we could end up in /**/** whitish is wrong, and if e.g. Xcode isn't checking anymore the package.swift we should do.

Reply:
I would not add a logic for a scenario we know is not valid. SPM's `exclude` only supports files or directories, not globs 🙏

Reply:
reverted